### ISSUE TITLE
Spam Filter

### ISSUE THREAD

**Issue by SeqSEE** [SeqSEE/discord-sentinel#3]
Spam filter for:
Emoji spam
All caps spam
Repetition spam
Bad Words List

**Comment by Thalma** [comment]
maybe: external link spam, or porn links! warn if they are publishing it on the wrong channel!

**Comment by SeqSEE** [comment]
> maybe: external link spam, or porn links! warn if they are publishing it on the wrong channel!

Yeah I agree something to help with that would be [UT1 Blacklists](https://github.com/olbat/ut1-blacklists)
